scapy/layers/l2.py

```diff
@@ -521,7 +521,10 @@ class Loopback(Packet):
     """*BSD loopback layer"""
 
     name = "Loopback"
-    fields_desc = [LoIntEnumField("type", 0x2, LOOPBACK_TYPES)]
+    if consts.OPENBSD:
+        fields_desc = [EnumField("type", 0, LOOPBACK_TYPES, "!I")]
```

Comment:
Is there a reason why
  - you choose a different default value?
  - you don't use an `IntEnumField`?

Thanks!

Reply:
The super class of LoIntEnumField is EnumField, there is no IntEnumField.
NameError: name 'IntEnumField' is not defined

I did not see the reason why the default is AF_INET.  When a packet is sniffed, no default is needed.  Sending IPv6 fails for some other reason.  I will debug this later.  If you think different defaults are confusing, I can change the default to 0x2.

Reply:
According to `layers/inet.py`, it doesn’t change much:
```
bind_layers(Loopback, IP, type=0)
bind_layers(Loopback, IP, type=2)
```
It would be better to keep it consistent, but except we want to add tests on a single `Loopback` layer with no bindings (the value with be overwritten whenever we add an upper layer), it’s no big deal.

Reply:
You need to import it:
```
$ git grep 'class IntEnumField'
scapy/fields.py:class IntEnumField(EnumField):
```

Reply:
I found the IntEnumField class.  LoIntEnumField does not use it for the some reason, so I made it consistently.  OpenBSD needs the address family in network byte order.   As far as I understand it, IntEnumField uses host byte order, there is no !I conversion.

Reply:
Of course, `IntEnumField` uses network byte order. The opposite would be surprising in a Network oriented tool such as Scapy, don't you think so?

I don't agree with "LoIntEnumField does not use it for the some reason" (or it is for a bad reason). If you want to make things consistently, I think you can replace:
```Python
class LoIntEnumField(EnumField):
    def __init__(self, name, default, enum):
        EnumField.__init__(self, name, default, enum, "!I")
    [...]
```

by

```Python
class LoIntEnumField(IntEnumField):
    [...]
```

Reply:
I have found where Field.__init__ adds the ! to the pack format string.  So I was confused that Scapy behaves differently than the generic Python pack format and LoIntEnumField ignored this Scapy optimization.
 
I have pushed an updated commit with your suggestions.